exercises/concept/stage-heralding/.meta/design.md

```diff
@@ -0,0 +1,50 @@
+# Design
+
+## Learning objectives
+
+### Primary objectives
+
+- Know what the `r` in `r""` means; in other words: know how to use string literals as a shortcut to define an instance of a type.
+- Know about the `r""` string literal.
```

Comment:
I'd say these are string macros, not literals. Or maybe they're regex literals.

Reply:
The manual calls them non-standard string literals: https://docs.julialang.org/en/v1/manual/metaprogramming/#Non-Standard-String-Literals

Reply:
Well they're a bit more authoritative than me :)